src/test/java/ticketservice/service/TicketServiceTest.java

```diff
@@ -0,0 +1,106 @@
+package ticketservice.service;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import ticketservice.model.Ticket;
+import ticketservice.model.TicketType;
+import ticketservice.model.User;
+import ticketservice.repository.TicketRepository;
+
+@ExtendWith(MockitoExtension.class)
+class TicketServiceTest {
+  @Mock private TicketRepository ticketRepository;
+
+  @InjectMocks private TicketService ticketService;
+
+  @Test
+  void Should_SaveTicket_When_CallSaveTicket() {
+    Ticket actualTicket = new Ticket();
+    Mockito.when(ticketRepository.save(actualTicket)).thenReturn(actualTicket);
+    Ticket resultTicket = ticketService.saveTicket(actualTicket);
+    assertEquals(actualTicket, resultTicket);
+    verify(ticketRepository, times(1)).save(actualTicket);
```

Comment:
use proper names:
1. expectedTicket instead of actualTicket
2. while resultTicket can be: actualTicket or resultTicket